src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/session/GetSessionsTest.kt

```diff
@@ -59,6 +59,40 @@ class GetSessionsTest(@Autowired private val objectMapper: ObjectMapper) : Integ
     assertSession(visitSessionResults[0], nextAllowedDay, sessionTemplate)
   }
 
+  @Test
+  fun `visit sessions are returned for enhanced prisoner a prison for a single schedule`() {
+    // Given
+    val prisonCode = "MDI"
+    val prisonerId = "A1234AA"
+
+    prisonApiMockServer.stubGetOffenderNonAssociationEmpty(prisonerId)
+    prisonApiMockServer.stubGetPrisonerDetails(prisonerId, prisonCode)
+
```

Comment:
I presume the call to hasPrisonerGotEnhancedPrivilege() will be added after my implementation.